src/event_ffi.mjs

```diff
@@ -1,3 +1,22 @@
 export function target(event) {
   return event.target;
 }
+
+export function constructorName(value) {
+  return value.constructor.name;
+}
+
+export function toKeyboardEventUnsafe(event) {
+  return {
+    event,
+    alt_key: event.altKey,
```

Comment:
I think it might be better to have super low functions. i.e.
```js
export function alt_key(event) {
  return event.altKey;
}
```
And let people build there own helpers on top of that

Reply:
I prefer if the event properties are fields instead of functions, and only the instance methods like [getModifierState](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState) are functions so it's more similar to the base api

Though as long as `alt_key` uses the signature `fn (KeyboardEvent) -> Bool` and not `fn (Event) -> Result(Bool, Nil)` I like that too

But in that case, we would need to add an `alt_key` function for each variant that has it. [TouchEvent](https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/altKey) also has `alt_key`. If we want to use the same function name `alt_key` and not name the functions `alt_key_keyboard` and `alt_key_touch` we would need to separate the functions to their own module like `event/keyboard.gleam` an `event/touch.gleam`